[2025-09-03 05:28:39] http://localhost
[2025-09-03 05:28:39] [Notice] Task process started.
[2025-09-03 05:28:55] [Error] Server error: `GET https://zenodo.org/api/communities/ror-data/records?q=&sort=newest` resulted in a `502 Bad Gateway` response:
<html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>ngin (truncated...)

[2025-09-03 05:28:55] [Notice] Task process stopped.
